.github/CONTRIBUTING.md

```diff
@@ -352,6 +353,64 @@ acceptance tests and run them for you on our side. This might mean that your PR
 takes a bit longer to merge, but it most definitely is not a blocker for
 contributions.
 
+#### Acceptance Tests With VCR
+
+The provider's acceptance tests are integrated with
+[go-vcr](https://github.com/dnaeon/go-vcr) to allow running, fast, valid, off
+the wire acceptance tests. The signal for VCR mode is the ENV var
+`OKTA_VCR_TF_ACC` with a value of `record` or `play`. If the mode is `play`
+then the named test is run for each of the cassettes (YAML files) in the
+`test/fixtures/vcr/{test-name}/` directory. If the mode is `record` then the
+ENV var `OKTA_VCR_CASSETTE` with a value for the cassette name must be set. All
+of the HTTP requests/responses are recorded to that cassette file. Finally, if
+the `test/fixtures/vcr/{test-name}/` directory or
+`test/fixtures/vcr/{test-name}/{cassete-name}`file does not exist then that
+named test is marked a skip.
+
+The harness will not overwrite an existing cassette file when it is in record
+mode.  The developer must delete that cassette from the file system first
+before it is re-recorded.
+
+An important subtlety to be called out here is that cassettes can be recorded
+for different orgs. Therefore using an intelligent naming convention for
+`OKTA_VCR_CASSETTE` new cassettes can be recorded an org with specific feature
+flags enabled on it. Thereafter, that org doesn't need to be live any further
+to have its tests run as they are replayed with VCR. If you add new cassettes
+to the repo please notes about them in the `test/fixtures/vcr/CASSETTES.md`
```

Comment:
My doco needs cleaned up "... please add notes ...".
Also, can you add a placeholder test/fixtures/vcr/CASSETTES.md file?